File: backend-api/src/main/java/com/smartenergy/backendapi/repository/RegistroConsumoRepository.java
package com.smartenergy.backendapi.repository;

import com.smartenergy.backendapi.model.RegistroConsumo;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface RegistroConsumoRepository extends JpaRepository<RegistroConsumo, UUID> {
}

File: mobile-app/app/src/main/java/com/example/smartenergy/repository/RegistroConsumoRepository.kt
package com.example.smartenergy.repository

class RegistroConsumoRepository {

    private val registros = mutableListOf<RegistroConsumo>()

    fun obtenerRegistros(): List<RegistroConsumo> {
        return registros
    }

    fun agregarRegistro(registro: RegistroConsumo) {
        registros.add(registro)
    }
}